api/src/State/PeriodPersistProcessor.php

```diff
@@ -94,7 +94,7 @@ public static function updateDaysAndScheduleEntries(Period $period, Period $orig
 
                 // Remove Days at end
                 for ($i = $daysCount; $i > $length; --$i) {
-                    $day = $days[$i - 1];
+                    $day = $days[max(0, $i - 1)];
```

Comment:
This should never occur. `$i` should not be 0. Actually `$i` should always be larger than 1.
Should we throw an error if `$i<=1` instead of silently move on?

Reply:
This would be the other way around.
https://github.com/ecamp/ecamp3/pull/3193/commits/9c0843bde0ec786fa60118bd6a97059b43f527a5

Reply:
Nevermind. I found some flaws in the logic of `PeriodPersistProcessor` and currently refactoring this code, anyway.

Reply:
ok, so i rebase and merge